src/bun.js/modules/Buffer.h

```diff
@@ -0,0 +1,27 @@
+#include "../bindings/ZigGlobalObject.h"
+#include "JavaScriptCore/JSGlobalObject.h"
+
+namespace Zig {
+
+inline void generateBufferSourceCode(JSC::JSGlobalObject* lexicalGlobalObject, JSC::Identifier moduleKey, Vector<JSC::Identifier, 4>& exportNames, JSC::MarkedArgumentBuffer& exportValues) {
+    JSC::VM& vm = lexicalGlobalObject->vm();
+    GlobalObject* globalObject = reinterpret_cast<GlobalObject*>(lexicalGlobalObject);
+
+    exportNames.append(JSC::Identifier::fromString(vm, "Buffer"_s));
```

Comment:
we should also add a default object which contains each of these properties in here

Reply:
We don't actually need that, adding a `exportNames.append(vm->propertyNames.defaultExportKey)` will add a `"default"` export to the module, sth like:
```
import buffer from "buffer";
console.log(buffer);
// {"default": ..., "buffer": ...}
```